api-catalog-services/src/test/java/org/zowe/apiml/apicatalog/health/ApiCatalogHealthIndicatorTest.java

```diff
@@ -17,32 +18,109 @@
 import org.springframework.cloud.client.discovery.DiscoveryClient;
 
 import java.util.Collections;
+import java.util.HashMap;
 
 import static org.junit.Assert.*;
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.when;
 
 public class ApiCatalogHealthIndicatorTest {
+    private static String AUTHENTICATION_SERVICE_ID = "apiml.authorizationService.zosmfServiceId";
+    private static String AUTHENTICATION_SERVICE_PROVIDER = "apiml.authorizationService.provider";
+    private static String ZOSMF = "zosmf";
+    private static String GATEWAY_SERVICE_ID = CoreService.GATEWAY.getServiceId();
 
     private final DiscoveryClient discoveryClient = mock(DiscoveryClient.class);
     private final ApiCatalogHealthIndicator apiCatalogHealthIndicator = new ApiCatalogHealthIndicator(discoveryClient);
     private final Health.Builder builder = new Health.Builder();
 
     @Test
-    public void testStatusIsUpWhenGatewayIsAvailable() {
-        when(discoveryClient.getInstances(CoreService.GATEWAY.getServiceId())).thenReturn(
-            Collections.singletonList(
-                new DefaultServiceInstance(CoreService.GATEWAY.getServiceId(), "host", 10010, true)));
+    public void testStatusIsUpWhenGatewayIsAvailableAndProviderIsAvailable() {
```

Comment:
Use  the `given_when_then` convention if possible

Reply:
My bad, I am on it.